Boa.Constrictor/WebDriver/Tasks/Check.cs

```diff
@@ -66,6 +67,16 @@ public override void PerformAs(IActor actor, IWebDriver driver)
             if (selectedState != CheckState) actor.AttemptsTo(Click.On(Locator));
         }
 
+        /// <summary>
+        /// Returns a description of the task.
+        /// </summary>
+        /// <returns></returns>
+        public override string ToString()
+        {
+            var action = CheckState == true ? "on" : "off";
+            return $" check {action} {Locator.Description}";
```

Comment:
Do you want that space at the beginning of the string?

Reply:
Oh, no. Good catch, thanks!